f8a_worker/dispatcher/flows/stackApiGraphV2Flow.yml

```diff
@@ -0,0 +1,14 @@
+---
+  flow-definitions:
+    - name: 'stackApiGraphV2Flow'
```

Comment:
Is it needed to keep v1 and v2 version in git? Can we directly change the existing version?